USAGE.md

```diff
@@ -197,6 +197,8 @@ Enable `Record user sessions` on the [PostHog project settings](https://us.posth
 
 [Authorized Domains for Replay](https://us.posthog.com/settings/project-replay#replay-authorized-domains) has to be disabled, for now, or create a new testing project to not disrupt your current project configuration.
 
+Let us know your `organization_id` so we can enable the `session-replay-mobile` feature flag for your organization.
```

Comment:
@pauldambra how do I know the `organization_id`? checked the project config but didn't find it.